streams/src/test/java/org/apache/kafka/streams/processor/internals/KeyValueStoreMaterializerTest.java

```diff
@@ -117,9 +117,12 @@ public void shouldCreateKeyValueStoreWithTheProvidedInnerStore() {
         final KeyValueStoreMaterializer<String, Integer> materializer = new KeyValueStoreMaterializer<>(materialized);
         final StoreBuilder<KeyValueStore<String, Integer>> builder = materializer.materialize();
         final KeyValueStore<String, Integer> built = builder.build();
-        final StateStore inner = ((WrappedStateStore) built).inner();
+        StateStore innerMost = built;
```

Comment:
I think it might be good to use unified variable naming for wrapping throughout the code base. I would suggest "wrapped" and "root" for a wrapped and most inner store. I would avoid "inner" as it seems to be ambiguous (could refer to wrapper or root).

Thoughts?

Reply:
This test was brittle. It shouldn't matter whether the Materializer wraps the given store or not. What's important is that it does actually use our supplier to make a store.

Reply:
Cannot follow?

Reply:
The test is verifying that the supplied store is actually used by the Materializer (as opposed to using some other store, I guess?)

To verify that, the test does not need to depend on any specific wrapping strategy. All it needs to do is be able to check that the supplied store is in the mix somehow. I think the given name is sufficient, but I guess the ultimate would be to insert something in the materialized result and then make sure it appears on the directly referenced store.